### PR TITLE
adding in notes for browser destinations

### PR DESCRIPTION
**Pull request by coreyching**
Added in a separate example for browser destinations when initializing your destination. 

_A summary of your pull request, including the what change you're making and why._

## Testing

_Include any additional information about the testing you have completed to
ensure your changes behave as expected. For a speedy review, please check
any of the tasks you completed below during your testing._

- [ ] Added [unit tests](https://github.com/segmentio/action-destinations/blob/main/docs/testing.md#local-end-to-end-testing) for new functionality
- [ ] Tested end-to-end using the [local server](https://github.com/segmentio/action-destinations/blob/main/docs/testing.md#local-end-to-end-testing)
- [ ] [Segmenters] Tested in the staging environment

### CODE REVIEW

File: docs/create.md [hunk 1]
```diff
@@ -111,6 +113,49 @@ With this minimal configuration, the destination can connect to the Segment App'
...
+    return window.yourSDKName
+  },
+
+  actions: {}
+}
+
+export default browserDestination(destination)
+```
+
+In Browser Destinations' no authentication is required. Instead, you must initialize your SDK with the required settings needed. To access your client throughout your destination,
```
Comment (danieljackins) [inline docs/create.md:153]:
```suggestion
In Browser Destinations' no authentication is required. Instead, you must initialize your SDK with the required settings. To access your client throughout your destination,
```

Comment (danieljackins) [reply docs/create.md:153]: "To access your client throughout your destination,"

I think something got cut off here

**Review by danieljackins — APPROVED** [review]

File: docs/create.md [hunk 2]
```diff
@@ -111,6 +113,49 @@ With this minimal configuration, the destination can connect to the Segment App'
...
+
+When importing your SDK, we recommend loading from a CDN when possible. This keeps the bundle size lower rather than directly including the SDK in our package.
+
+Make sure to add a global declarition where you specify your SDK as a field of a Window interface so you can reference and return it in your intialize function. E.g. (See above)
```
Comment (mshwery) [inline docs/create.md:157]:
```suggestion
Make sure to add a global declaration where you specify your SDK as a field of a Window interface so you can reference and return it in your initialize function. E.g. (See above)
```

**Review by mshwery — APPROVED** [review]

File: docs/create.md (hunk 1 above)
Comment (coreyching) [reply docs/create.md]: Oh good catch. That was a typo. Removed.

**Pull request merged**